packages/cmf/src/cmfConnect.md

```diff
@@ -221,6 +221,32 @@ If you want to render some component conditionally, just pass "renderIf" prop (t
 You can also use Expression for this and customize this prop like "renderIfExpression" in
 CMF json configuration files
 
+## How to access state and mute from the outside
```

Comment:
hi, by 'mute' do you mean 'mutate'?

Reply:
i think that is what he mean, but it is also misleading since the state is not mutated but updated with a non mutable operation.
`## Read and update component state from outside`

Reply:
👍